lib/src/values/logic_value.dart

```diff
@@ -552,9 +553,9 @@ abstract class LogicValue implements Comparable<LogicValue> {
   String toString({bool includeWidth = true}) {
     if (isValid && includeWidth) {
       // for ==_INT_BITS, still use BigInt so we don't get negatives
-      final hexValue = width >= _INT_BITS
+      final hexValue = width > _INT_BITS
           ? toBigInt().toUnsigned(width).toRadixString(16)
-          : toInt().toRadixString(16);
+          : toBigInt().toRadixString(16);
```

Comment:
what's the point of having different logic for different widths here then?  Maybe just reverting to `width >= _INT_BITS` is sufficient so that for 63 and less we dont have to build a BigInt?

Reply:
This is string output, so likely performance is not a big issue. if the width==64, the sign gets output, when I don't think we want that, we want to see the raw bits in radix form.

Reply:
This code seems redundant and simple to optimize.  Since `toBigInt()` in all cases returns an unsigned `BigInt` now (there's a `toUnsigned` beforehand for all three cases), then both of these lines are effectively the same:
```dart
toBigInt().toUnsigned(width).toRadixString(16)
toBigInt().toRadixString(16)
```

While performance for simulation is not impacted by this, it is relevant for SystemVerilog generation (constants), which is somewhat performance sensitive, although much less so.

I guess what I'm saying is one of these two options makes more sense to me:

Option 1: (only extra perf impact for 64 bit, since >64 already must do the `toBigInt`)
```dart
      final hexValue = width >= _INT_BITS // include 64-bit
          ? toBigInt().toRadixString(16)
          : toInt().toRadixString(16);
```

Option 2: (simpler solution, always perf impact)
```dart
      final hexValue = toBigInt().toRadixString(16);
```

I think option 1 is better?

Reply:
I tested version 1and pushed.

Reply:
Did the push go through?  Still looks the same?

Reply:
I just pushed the toString performance fix (not converting to BigInt unless >= _INT_MAX.
This case is special (I guess I should comment on it) because we really DO want to convert to BigInt at 64 in this case only to ignore the sign.